### ISSUE TITLE
Detailslist: Keyboard users are not able to adjust the width of columns

### ISSUE THREAD

**Issue by lezhang1** [microsoft/fluentui#17346]
### Environment Information

- **Package version(s)**: 7.154.0
- **Browser and OS versions**: Edge latest on Win 10

### Describe the issue:

Keyboard users are not able to adjust the width of columns of the details list. Keyboard users may find it difficult to adjust the column's width when large information is present in any of the row.

### Please provide a reproduction of the issue in a codepen:

you can just try this https://developer.microsoft.com/en-us/fluentui#/controls/web/detailslist/basic

#### Actual behavior:

Keyboard users are not able to adjust the width of columns of the details list.

#### Expected behavior:

Keyboard users should be able to adjust the column's width for better visual of row's information in the grid.

### Documentation describing expected behavior

There are multiple ways to do that, example we can use Ctrl+ Arrow up to adjust

**Comment by gouttierre** [comment]
@lezhang1 - thanks for the feedback. I see the behavior reported in Edge and Chrome. 

@khmakoto - Would you be able to confirm if this is a regression, or if this behavior is an issue? Are the keyboard users meant to adjust the columns dimensions using multi-key selections? 
cc @smhigley
